.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: php
 
+env:
+    global:
+        - SYMFONY_DEPRECATIONS_HELPER: weak
```

Comment:
why adding this line? it will hide any deprecation which means you could miss some changes required for Symfony 3.0